lib/bing-ads-reporting/insight_service.rb

```diff
@@ -42,35 +49,10 @@ def report_operation(options)
 
     def generate_report_message(options)
       period = options[:period]
-      report_type = options[:report_type]
-
-      report_request(period, options, report_type)
-    end
-
-    def report_request(period, options, _report_type)
-      {
-        'EntityType' => options[:entity_type],
-        'SearchParameters' => {
-          'SearchParameter' =>  date_range(period)
-        },
-        '@xsi:type' => 'tns:DateRangeSearchParameter'
-      }
-    end
-
-    def date_range(period)
-      {
-        'EndDate' => scope_data_range(period.to),
-        'StartDate' => scope_data_range(period.from),
-        '@i:type' => 'tns:DateRangeSearchParameter'
-      }
-    end
+      # report_type = options[:report_type]
```

Comment:
I'd remove these comments if they're not needed

Reply:
done.